src/ctime.js

```diff
@@ -50,6 +50,33 @@ export function date(val) {
     return date;
 }
 
+/**
+ * Resets our cTime to the beginning of a given unit
+ * @param {string} unitString to be set, unix-timestamp, native date object
+ * @returns {object} date instance
+ */
+export function startOf(t, unit) {
+    const d = date(t);
+    switch (unit) {
```

Comment:
https://github.com/cckrause/ctime/blob/master/src/ctime.js#L64

you reuse the given `set` interface

```suggestion
    d.set(0, unit);
```

Reply:
month would be a sepcial case though

Reply:
Did use the switch at that point because it would reset the given unit and all smaller units.
If `seconds` gets to be reset, it also resets the `milliseconds` and so on

Month should be okay since the first day is alway 01. It would be special if we implement 'endOf'